lib/helpers/coverage.js

```diff
@@ -9,73 +9,74 @@ const buildTable = require('./coverage-table');
 let coverage;
 let schema;
 
-module.exports.init = ({ reportCoverage, apiDefinitionsPath, buildSchemaOptions, printCoverageToFile }) => {
-    coverage = {};
-    const buildSchemaOptionsOverride = buildSchemaOptions || {};
-    schema = schemaUtils.getSchema(apiDefinitionsPath, buildSchemaOptionsOverride);
+module.exports.init = ({
+  reportCoverage, apiDefinitionsPath, buildSchemaOptions, exportCoverage,
+}) => {
+  coverage = {};
+  const buildSchemaOptionsOverride = buildSchemaOptions || {};
+  schema = schemaUtils.getSchema(apiDefinitionsPath, buildSchemaOptionsOverride);
 
-    if (reportCoverage === true) {
-        process.on('beforeExit', () => {
-            const parsedCoverage = getReport();
-            printReport(parsedCoverage);
-        });
-    }
-    if (printCoverageToFile === true) {
-        process.on('beforeExit', () => {
-            const parsedCoverage = getReport();
-            printReportToFile(parsedCoverage);
-        });
-    }
+  if (reportCoverage === true) {
+    process.on('beforeExit', () => {
+      const parsedCoverage = getReport();
```

Comment:
add another event listener for getting the report once

Reply:
done